command/sub8_missions/nodes/tp.yaml

```diff
@@ -0,0 +1,6 @@
+bin: -24 14 -3.5
```

Comment:
This file should go under sub8_gazebo/config. It should also be called teleport_checkpoints or something like that rather than tp.